File: Search and Sort/BubbleSort.py
def bubbleSort(array):
    for i in range(len(array)):
        for j in range(len(array) - i - 1):
            if (array[j] > array[j + 1]):
                array[j], array[j + 1] = array[j + 1], array[j]

if __name__ == "__main__":
    array = [ 64, 34, 25, 12, 22, 11, 90 ]
    
    bubbleSort(array)
    print(array)

File: Search and Sort/BubbleSort.cpp
#include <cstdio>
#include <algorithm>
using namespace std;

void bubbleSort(int array[], int length) {
    for (int i = 0; i < length; i++) {
        for (int j = 0; j < length - i - 1; j++) {
            if (array[j] > array[j + 1])
                swap(array[j], array[j + 1]);
        }
    }
}

int printArray(int array[], int length) {
    for (int i = 0; i < length; i++) {
        printf("%d ", array[i]);
    }
}

int main(int argc, char **argv) {
    int array[] = { 64, 34, 25, 12, 22, 11, 90 };
    int length = sizeof(array)/sizeof(int);

    bubbleSort(array, length);
    printArray(array, length);

    return 0;
}